jetty-maven-plugin/src/it/jetty-run-mojo-multi-module-single-war-it/postbuild.groovy

```diff
@@ -20,7 +20,7 @@ File buildLog = new File( basedir, 'build.log' )
 assert buildLog.text.contains( 'Started Jetty Server' )
 
 assert buildLog.text.contains( '(1a) >> javax.servlet.ServletContextListener loaded from jar:' )
-assert buildLog.text.contains( 'local-repo/javax/servlet/javax.servlet-api/3.1.0/javax.servlet-api-3.1.0.jar!/javax/servlet/ServletContextListener.class << (1b)' )
+assert buildLog.text.contains( 'javax/servlet/javax.servlet-api/3.1.0/javax.servlet-api-3.1.0.jar!/javax/servlet/ServletContextListener.class << (1b)' )
```

Comment:
Is this going to work in jetty-10 where we use a different version of the servlet api?

Reply:
it's already different in jetty-10 and the person who will merge to jetty-10 will have to take care but I guess it will be me :)